rest/model/task.go

```diff
@@ -388,33 +388,34 @@ func (at *APITask) GetProjectIdentifier() {
 // ToService returns a service layer task using the data from the APITask.
 func (ad *APITask) ToService() (interface{}, error) {
 	st := &task.Task{
-		Id:                  utility.FromStringPtr(ad.Id),
-		Project:             utility.FromStringPtr(ad.ProjectId),
-		Version:             utility.FromStringPtr(ad.Version),
-		Revision:            utility.FromStringPtr(ad.Revision),
-		Priority:            ad.Priority,
-		Activated:           ad.Activated,
-		ActivatedBy:         utility.FromStringPtr(ad.ActivatedBy),
-		BuildId:             utility.FromStringPtr(ad.BuildId),
-		DistroId:            utility.FromStringPtr(ad.DistroId),
-		Container:           utility.FromStringPtr(ad.Container),
-		BuildVariant:        utility.FromStringPtr(ad.BuildVariant),
-		DisplayName:         utility.FromStringPtr(ad.DisplayName),
-		HostId:              utility.FromStringPtr(ad.HostId),
-		Execution:           ad.Execution,
-		RevisionOrderNumber: ad.Order,
-		Status:              utility.FromStringPtr(ad.Status),
-		DisplayStatus:       utility.FromStringPtr(ad.DisplayStatus),
-		TimeTaken:           ad.TimeTaken.ToDuration(),
-		ExpectedDuration:    ad.ExpectedDuration.ToDuration(),
-		GenerateTask:        ad.GenerateTask,
-		GeneratedBy:         ad.GeneratedBy,
-		DisplayOnly:         ad.DisplayOnly,
-		Requester:           utility.FromStringPtr(ad.Requester),
-		CanSync:             ad.CanSync,
-		HasCedarResults:     ad.HasCedarResults,
-		CedarResultsFailed:  ad.CedarResultsFailed,
-		MustHaveResults:     ad.MustHaveResults,
+		Id:                      utility.FromStringPtr(ad.Id),
+		Project:                 utility.FromStringPtr(ad.ProjectId),
+		Version:                 utility.FromStringPtr(ad.Version),
+		Revision:                utility.FromStringPtr(ad.Revision),
+		Priority:                ad.Priority,
+		Activated:               ad.Activated,
+		ActivatedBy:             utility.FromStringPtr(ad.ActivatedBy),
+		BuildId:                 utility.FromStringPtr(ad.BuildId),
+		DistroId:                utility.FromStringPtr(ad.DistroId),
+		Container:               utility.FromStringPtr(ad.Container),
+		BuildVariant:            utility.FromStringPtr(ad.BuildVariant),
+		BuildVariantDisplayName: utility.FromStringPtr(ad.BuildVariantDisplayName),
```

Comment:
Should also probably add to BuildFromService

Reply:
It's actually already there